nulib/meadow
# Constrain select form elements

# Summary 
Constrain select form elements to a max-width to very long option titles don't interfere with surrounding UI

# Specific Changes in this PR
- Put some CSS on `select` elements to constrain width

# Version bump required by the PR

See [Semantic Versioning 2.0.0](https://semver.org/) for help discerning which is required.


# Steps to Test
1. Select a few items to batch edit
2. On Batch edit screen , go to Administrative tab and notice the "Collection" select menu is a reasonable width and not pushing the Project form elements to the right, off the screen.

Also please let developers know if there are any special instructions to test this in the development environment. 

# :rocket: Deployment Notes

**Note** - __if you check any of these boxes go to the [(always open) `main` <- `staging` PR](https://github.com/nulib/meadow/pulls) and add detailed notes and instructions to help out others who may end up deploying your changes to production__

- Backward compatible API changes
- Backwards-incompatible API changes


# Tested/Verified
